docs/docs/telemetry/events.json

```diff
@@ -57,6 +57,14 @@
           "type": "integer",
           "description": "Number of templates or responses defined in the domain."
         },
+        "num_conditional_response_variations": {
```

Comment:
it just occurred to me that we also have to update `telemetry.mdx`  (I think yours is not the first one to be missing there 😬 )

Reply:
Np, I'll add the properties after [L96 ](https://github.com/RasaHQ/rasa/blob/main/docs/docs/telemetry/telemetry.mdx)- is that the only update I have to make?

Reply:
yep 👍🏻 Maybe have a quick look if something else is missing